lib/configGenerator/generateAnsibleConfig.js

```diff
@@ -1,13 +1,43 @@
 const fs = require('fs').promises;
 
 const { PrivateKey } = require('@dashevo/dashcore-lib');
+const { Wallet } = require('@dashevo/wallet-lib');
 
 const BlsSignatures = require('bls-signatures');
 
 const yaml = require('js-yaml');
 
 const crypto = require('crypto');
 
+const { contractId: dpnsContractId, ownerId: dpnsOnwerId } = require('@dashevo/dpns-contract/lib/systemIds');
```

Comment:
You don't need this IDs and dependencies what you added in package.json anymore.

Reply:
Huh, Djavid went through with me and added these... it wasn't there before the PR!